README.md

```diff
@@ -14,19 +14,29 @@ Basic Configuration in the `mkdocs.yml` file:
-| file_types_to_check          | ["png", "jpg", "jpeg", "gif", "pdf", "ico", "drawio", "tif", "tiff", "zip", "tar.gz", "rar", "ogg", "mp3", "mp4", "vtt ", "ogv", "mov", "svg", "pot", "potx", "ppsx", "ppt", "pptx", "xlt", "xltx", "xls", "xlsx", "doc", "docx", "dot", "dotx", "vst", "vstx", "vsd", "vsdx"] | Only check these file types for their usage and delete them if necessary.                                                                                                                      |
-| file_names_to_never_remove   | ["favicon"]                                                                                                                                                                                                                                                                    | Files with these names will never get deleted, even if no usage is detected.                                                                                                                   |
-| folders_to_never_remove_from | ["assets"]                                                                                                                                                                                                                                                                     | Files in these folders will never get deleted, even if no usage is detected. Useful for always including specific static files.                                                                |
+| Setting                      | Default     | Description                                                                                                                                                                                                                    |
+|------------------------------|-------------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| enabled                      | True        | Whether the plugin is enabled when building your project. If you want to switch the plugin off, e.g. for local builds, use an [environment variables](https://www.mkdocs.org/user-guide/configuration/#environment-variables). |
+| enabled_on_serve             | False       | Whether the plugin is enabled when serving your project.                                                                                                                                                                       |
```

Comment:
I'm asking myself if `enabled_on_serve` would be a more "sane" default to prevent accidental leakage of unused assets by users that haven't read this plugin's documentation thoroughly, expecting the plugin to wipe these unused files away.

Open for input

Reply:
IMHO, no, because serve in mkdocs is used only for local development purpose, and locally you do not care about orphaned files - its imported on the final build that goes for publishing.
Checking orphaned files on serve, just extends local development unnecessary. If someone really wanto to do it, then can enable it explicitly - and this is a purpose of `enabled_on_serve`.
- ... `read this plugin's documentation` ... actually it's a user problem ;) because its documented opposite to "no info in the docs at all".

Reply:
just added more descriptive explanation to the docs.